paper/paper.bib

```diff
@@ -1,13 +1,178 @@
+and Alekseyev, Max A.
+and Pevzner, Pavel A.},
+  Editor                   = {Deng, Minghua
+and Jiang, Rui
+and Sun, Fengzhu
+and Zhang, Xuegong},
+  Pages                    = {158--170},
+  Publisher                = {Springer Berlin Heidelberg},
+  Year                     = {2013},
+
+  Address                  = {Berlin, Heidelberg},
+
+  Abstract                 = {Recent advances in single-cell genomics provide an alternative to gene-centric metagenomics studies, enabling whole genome sequencing of uncultivated bacteria. However, single-cell assembly projects are challenging due to (i) the highly non-uniform read coverage, and (ii) a greatly elevated number of chimeric reads and read pairs. While recently developed single-cell assemblers have addressed the former challenge, methods for assembling highly chimeric reads remain poorly explored. We present algorithms for identifying chimeric edges and resolving complex bulges in de Bruijn graphs, which significantly improve single-cell assemblies. We further describe applications of the single-cell assembler SPAdes to a new approach for capturing and sequencing ``dark matter of life'' that forms small pools of randomly selected single cells (called a mini-metagenome) and further sequences all genomes from the mini-metagenome at once. We demonstrate that SPAdes enables sequencing mini-metagenomes and benchmark it against various assemblers. On single-cell bacterial datasets, SPAdes improves on the recently developed E+V-SC and IDBA-UD assemblers specifically designed for single-cell sequencing. For standard (multicell) datasets, SPAdes also improves on A5, ABySS, CLC, EULER-SR, Ray, SOAPdenovo, and Velvet.},
+  Booktitle                = {Research in Computational Molecular Biology: 17th Annual International Conference, RECOMB 2013, Beijing, China, April 7-10, 2013. Proceedings},
+  Doi                      = {10.1007/978-3-642-37195-0_13},
+  ISBN                     = {978-3-642-37195-0},
+  Url                      = {https://doi.org/10.1007/978-3-642-37195-0_13}
+}
+
+@Article{sra2008,
+  Title                    = {Database resources of the National Center for Biotechnology Information},
+  Author                   = {Wheeler, David L. and Barrett, Tanya and Benson, Dennis A. and Bryant, Stephen H. and Canese, Kathi and Chetvernin, Vyacheslav and Church, Deanna M. and DiCuccio, Michael and Edgar, Ron and Federhen, Scott and Feolo, Michael and Geer, Lewis Y. and Helmberg, Wolfgang and Kapustin, Yuri and Khovayko, Oleg and Landsman, David and Lipman, David J. and Madden, Thomas L. and Maglott, Donna R. and Miller, Vadim and Ostell, James and Pruitt, Kim D. and Schuler, Gregory D. and Shumway, Martin and Sequeira, Edwin and Sherry, Steven T. and Sirotkin, Karl and Souvorov, Alexandre and Starchenko, Grigory and Tatusov, Roman L. and Tatusova, Tatiana A. and Wagner, Lukas and Yaschenko, Eugene},
+  Journal                  = {Nucleic Acids Research},
+  Year                     = {2008},
```

Comment:
Why exactly the publication of 2008? A similar article is published every year, most recently https://doi.org/10.1093/nar/gkw1071 
Or is there a specific reason I am missing?

Reply:
wilco